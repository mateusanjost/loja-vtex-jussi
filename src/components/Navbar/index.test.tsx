import TestRenderer from "react-test-renderer";
import NavbarComponent from "./index";

describe("Navbar", () => {
  it("should match snapshot", () => {
    const testRenderer = TestRenderer.create(<NavbarComponent />);
    const jsonComponent = testRenderer.toJSON();

    expect(jsonComponent).toMatchSnapshot();
  });
});
