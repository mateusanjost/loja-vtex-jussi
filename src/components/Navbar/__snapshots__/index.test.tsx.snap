// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should match snapshot 1`] = `
<nav
  className="sc-bdnxRM gBOXHT"
>
   
  <div
    className="sc-dlnjwi bmkcnO"
  >
    <a
      href="https://jussi.com.br/services.html"
      rel="noopener noreferrer"
      target="_blank"
    >
      Nossas soluções
    </a>
    <a
      href="https://jussi.com.br/"
      rel="noopener noreferrer"
      target="_blank"
    >
      Conheça a Jüssi
    </a>
  </div>
  <div
    className="sc-hKFxyN gLWLBP"
  >
    <input
      className="sc-gtsrHT jVDMiC"
      onChange={[Function]}
      onKeyPress={[Function]}
      placeholder="Buscar"
      type="text"
    />
    <div
      className="sc-eCApnc hmDsgX"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        <a>
          <div>
             Login
          </div>
        </a>
      </a>
      <img
        alt="Shopping Cart - Jussi"
        src="shopping-cart.svg"
      />
    </div>
  </div>
</nav>
`;
